sklearn/metrics/classification.py

```diff
@@ -2038,3 +2038,60 @@ def brier_score_loss(y_true, y_prob, sample_weight=None, pos_label=None):
     y_true = np.array(y_true == pos_label, int)
     y_true = _check_binary_probabilistic_predictions(y_true, y_prob)
     return np.average((y_true - y_prob) ** 2, weights=sample_weight)
+
+
+def calibration_loss(y_true, y_prob, bin_size=2):
+    """Compute Calibration score by bins.
+    The calibration loss is defined as the measure to access the quality of
+    learning methods and learned models. A calibration measure based on
+    overlaping binning is CAL (Caruana and Niculescu-Mizil, 2004).
+
+    Parameters
+    ----------
+    y_true : array, shape (n_samples,)
+        True targets.
+     y_prob : array, shape (n_samples,)
+        Probabilities of the positive class.
+     bin_size : int
+        Size of the bin (samples) analysed in each iteration
+     Returns
```

Comment:
Try to the let a newline below returns and try to `---` match the length of `Return`

```suggestion

     Returns
     ----------
```